src/AI.js

```diff
@@ -115,6 +115,32 @@ function blockPath() {
 }
 
 function main() {
+  //Pause game stuff ////////////
+  $(document).bind("keydown", function(e){
+    console.log(e);
```

Comment:
you should remove this debugging statement before we merge so everyone doesn't have a much of logs in their console.